core/controllers/question_editor.py

```diff
@@ -199,25 +313,18 @@ def get(self, question_id):
         self.render_json(self.values)
 
     @acl_decorators.can_edit_question
-    def put(self, question_id):
+    def put(self, question_id: str) -> None:
         """Updates properties of the given question."""
-        commit_message = self.payload.get('commit_message')
+        assert self.user_id is not None
+        assert self.normalized_payload is not None
+        commit_message = self.normalized_payload.get('commit_message')
+        change_list = self.normalized_payload.get('change_list')
 
-        if not commit_message:
+        if commit_message is None:
```

Comment:
This can be disallowed by schema, right?

Reply:
Fixed